### PR TITLE
fix condition for venv

### PR DESCRIPTION
**Pull request by shubhras01**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -84,7 +84,7 @@ check-code:
 
 .PHONY: env
 env: clean
-	if [[ "$$VIRTUAL_ENV" == "" ]];\
+	if [[ "$$VIRTUAL_ENV" != "" ]];\
```
Comment (angrybayblade) [inline Makefile:87]: @shubhras01 We should not create a new env while we're already in a virtual environment, that's why this checks verifies if we're already in a virtual environment or not take a look here - https://docs.python.org/3/library/venv.html#module-venv

Comment (shubhras01) [reply Makefile:87]: yes, correct me if i am wrong - after this check we clean the existing env. And `if [[ "$$VIRTUAL_ENV" == "" ]];` condition means if venv does not exist.

Comment (angrybayblade) [reply Makefile:87]: Yes, that is correct

Comment (shubhras01) [reply Makefile:87]: so umm, why are we cleaning the venv if venv doesnt exist ?

Comment (angrybayblade) [reply Makefile:87]: Sorry there's a small confusion here, The environment variable `$VIRTUAL_ENV` contains path to the activated environment. If the value of `$VIRTUAL_ENV` is empty that does not mean the environment does not exist it means the environment is not activated. It still exists that's why we're running `pipenv --rm` and `pipenv --clear`

**Pull request closed without merging**